### PR TITLE
[stable/wordpress] fix required string value being parsed as bool

### PR DESCRIPTION
**Pull request by prydonius**
Reverts #7101 which breaks ensuring the allowEmptyPassword is always a
string of value "yes" or "no".

### CODE REVIEW

File: stable/wordpress/templates/deployment.yaml [hunk 1]
```diff
@@ -28,7 +28,11 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
         env:
         - name: ALLOW_EMPTY_PASSWORD
-          value: {{ .Values.allowEmptyPassword | quote }}
+        {{- if .Values.allowEmptyPassword }}
```
Comment (davidkarlsen) [inline stable/wordpress/templates/deployment.yaml:31]: won't "no" be evaluated as true as well? https://github.com/helm/helm/blob/master/docs/chart_template_guide/control_structures.md

File: stable/wordpress/values.yaml [hunk 2]
```diff
@@ -55,7 +55,7 @@ wordpressTablePrefix: wp_
 
 ## Set to `yes` to allow the container to be started with blank passwords
 ## ref: https://github.com/bitnami/bitnami-docker-wordpress#environment-variables
-allowEmptyPassword: yes
+allowEmptyPassword: "yes"
```
Comment (juan131) [inline stable/wordpress/values.yaml]: It's enough with this change. You don't need to revert the rest of the changes.

**Review by juan131 — CHANGES_REQUESTED** [review]

File: stable/wordpress/values.yaml (hunk 2 above)
Comment (prydonius) [reply stable/wordpress/values.yaml]: Yes, but keeping the if/else ensures that if someone sets this from the CLI (e.g. `helm install wordpress --set allowEmptyPassword=yes`) and if Helm accidentally treats that as boolean this still works.

File: stable/wordpress/templates/deployment.yaml (hunk 1 above)
Comment (prydonius) [reply stable/wordpress/templates/deployment.yaml:31]: @davidkarlsen you're right, what if we change the Helm value to actually be a boolean (works well with `--set`), and then make sure the template always renders "yes" or "no" strings depending on the value of the boolean?

Comment (davidkarlsen) [reply stable/wordpress/templates/deployment.yaml:31]: Either that or just render the literal value from values.yaml, documenting it should be "yes" || "no" - I'm good with either option.

**Pull request merged**